Dockerfile.armhf

```diff
@@ -17,6 +17,10 @@
 
 FROM armhf/debian:jessie
 
+# Allow replacing httpredir mirror
+ARG APT_MIRROR=httpredir.debian.org
+RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
```

Comment:
Yes, i can do that.

I've confirmed there is armhf for jessie on `deb.debian.org`
https://deb.debian.org/debian/dists/jessie/main/